docs/source/howto/data.rst

```diff
@@ -260,6 +260,188 @@ Finding data
+    Group type_string  user
+    Group description  <no description>
+    -----------------  ----------------
+    # Nodes:
+    PK    Type    Created
+    ----  ------  ---------------
+     1    Code    26D:21h:45m ago
+
+
+Remove nodes from a Group
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+From the command line interface::
+
+    verdi group remove-nodes -G test_group 1
+    Do you really want to remove 1 nodes from Group<test_group>? [y/N]: y
+
+From the python interface::
+
+    In [1]: group = Group.get(label='test_group')
+    In [2]: group.clear()
+
+
+Rename Group
+^^^^^^^^^^^^
+
+From the command line interface::
+
+      verdi group relabel test_group old_group
+      Success: Label changed to old_group
+
+From the python interface::
+
+    In [1]: group = Group.get(label='old_group')
+    In [2]: group.label = "another_group"
+
+
+Delete Group
+^^^^^^^^^^^^
+
+From the command line interface::
+
+      verdi group delete another_group
+      Are you sure to delete Group<another_group>? [y/N]: y
+      Success: Group<another_group> deleted.
+
+
+Copy one group into another
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This operation will copy the nodes of the source group into the destination
+group. Moreover, if the destination group did not exist before, it will
+be created automatically.
+
+From the command line interface::
+
+    verdi group copy source_group dest_group
+    Success: Nodes copied from group<source_group> to group<dest_group>
+
+From the python interface::
+
+    In [1]: src_group = Group.objects.get(label='source_group')
+    In [2]: dest_group = Group(label='destination_group').store()
+    In [3]: dest_group.add_nodes(list(src_group.nodes))
+
+
+Create a `Group` subclass
+-------------------------
+It is possible to create a subclass of `Group` to implement custom functionality.
+To make the instances of the subclass storable and loadable, it has to be registered through an entry point in the ``aiida.groups`` entry point category.
+For example, assuming we have a subclass ``SubClassGroup`` in the module ``aiida_plugin.groups.sub_class:SubClassGroup``, to register it, one has to add the following to the ``setup.py`` of the plugin package::
+
+    "entry_points": {
+        "aiida.groups": [
+            "plugin.sub_class = aiida_plugin.groups.sub_class:SubClassGroup"
+        ]
+    }
+
+Now that the subclass is properly registered, instances can be stored::
+
+    group = SubClassGroup(label='sub-class-group')
+    group.store()
+
+The ``type_string`` of the group instance corresponds to the entry point name and so in this example is ``plugin.sub_class``.
+This is what AiiDA uses to load the correct class when reloading the group from the database::
+
+    group = load_group(group.pk)
+    assert isinstance(group, SubClassGroup)
+
+If the entry point is not currently registered, because the corresponding plugin package is not installed for example, AiiDA will issue a warning and fallback onto the ``Group`` base class.
+
+
+
```

Comment:
This is part of another issue and I will take care of this. Please remove it here.

Reply:
removed